paasta_tools/cli/cmds/status.py

```diff
@@ -464,6 +467,33 @@ def print_kubernetes_status(
     return 0
 
 
+def print_tron_status(
+    service: str,
+    instance: str,
+    output: List[str],
+    tron_status,
+    verbose: int = 0,
+) -> int:
+    output.append(f"    Tron job: {tron_status.job_name}")
+    if verbose:
+        output.append(f"      Status: {tron_status.job_status}")
+        output.append(f"      Schedule: {tron_status.job_schedule}")
+    output.append("      DashBoard: {}".format(PaastaColors.blue(tron_status.job_url)))
+
+    output.append(f"    Action: {tron_status.action_name}")
+    output.append(f"      Status: {tron_status.action_state}")
+    if verbose:
+        output.append(f"      Start time: {tron_status.action_start_time}")
+    output.append(f"      Command: {tron_status.action_command}")
+    if verbose:
+        output.append(f"      Raw Command: {tron_status.action_raw_command}")
+        output.append(f"      Stdout: {tron_status.action_stdout}")
```

Comment:
I think this should only show on two v's. Also put a newline before the stdout because it is several lines.

Reply:
Raw command should be with verbose == 1.  Also, can you also add a new line after the ":" so it looks like the following?
```
Stdout:
logging.info ...
logging.debug ...

Stderr:
Exception!
```